ports/libeventheader-tracepoint/portfile.cmake

```
@@ -6,14 +6,13 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO "microsoft/LinuxTracepoints"
     REF "v${VERSION}"
-    SHA512 b296ad3ee102d45cd8bccb2e3ed478f3d7adff8b3650251926189fd6efbca38728db61208af1627c08c16641b349e31e9366c6bc1965795063f39a167181f067
+    SHA512 bac090cdcf42fea2bc5da973df15a148b314c3c0b1183a571968aa64517c4fad52662c45a6e636681588a51cbf0807ad3235c3dee5d678e9e4f48c52fc5cbf04
     HEAD_REF main)
 
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}/libeventheader-tracepoint"
     OPTIONS
-        -DBUILD_SAMPLES=OFF
-        -DBUILD_TOOLS=OFF)
+        -DBUILD_SAMPLES=OFF)

```

```suggestion
        -DBUILD_SAMPLES=OFF
)
```

And here needs to explain why removed the option `BUILD_TOOLS=OFF`

---

Removed because the option is no longer present in the project's CMakeFile.txt.